data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,802 @@
+# Copyright (c) 2023 by Azure
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+
+# Kinetic Weapons:
+
+outfit "Speck Round"
+	category "Ammunition"
+	cost 6
+	thumbnail "outfit/speck round"
+	"mass" 0.003
+	"speck capacity" -1
+	"flotsam chance" 1
+	description "These densely wound metallic glass slugs are capable of inflicting a good amount of kinetic damage via hypervelocity impacts, but are easily stopped by momentum-repulsion shield systems."
+
+
+outfit "Speck Magazine"
+	category "Ammunition"
+	cost 60000
+	thumbnail "outfit/unknown"
+	"mass" 2.4
+	"outfit space" -6
+	"speck capacity" 1200
+	description "While Speckle Coilguns carry a considerable amount of ammunition in their internal magazines, for longer expeditions or for resupply operations the Avgi use these compact ammunition storages."
+
+
+outfit "Speckle Coilgun"
+	category "Guns"
+	cost 40000
+	thumbnail "outfit/speckle coilgun"
+	"mass" 6.3
+	"outfit space" -9
+	"weapon capacity" -9
+	"energy capacity" 90
+	"gun ports" -1
+	"speck capacity" 900
+	weapon
+		sound "fleck"
+		ammo "Speck Round"
+		"fire effect" "speckle fire"
+		"inaccuracy" 1.2
+		"velocity" 60
+		"lifetime" 0
+		"reload" 20
+		"burst reload" 15
+		"burst count" 3
+		"cluster"
+		"firing energy" 36
+		"firing heat" 30
+		"firing force" 12
+		"submunition" "Speck"
+	description "This compact coilgun fires a stream of hypervelocity projectiles, accelerating them on a traveling wave of magnetic energy as capacitors are discharged through a series of coils. Designed primarily for drones and smaller ships, the Speckle is a deadly albeit relatively unsophisticated weapon."
+	description "Due to its limited range, the Speckle shines when a ship can sneak close enough to an enemy to bring it to bear, where it can begin tearing through armor and hull. However, its effectiveness is much more limited against shields, which shrug off the its mass driver rounds with relative ease."
```

Comment:
```suggestion
	description "Due to its limited range, the Speckle shines when a ship can sneak close enough to an enemy to bring it to bear, where it can begin tearing through armor and hull. However, its effectiveness is much more limited against shields, which shrug off its mass driver rounds with relative ease."
```